flare_portal/experiments/templates/experiments/participant_list.html

```diff
@@ -53,6 +64,23 @@ <h3 class="card-title">Participants</h3>
                                 <tbody>
                                     {% for form in formset %}
                                         <tr>
+                                            <td>
+                                                <label class="form-label mt-2">
+                                                    <input
+                                                        class="form-check"
+                                                        type="checkbox"
+                                                        aria-label="Press to select this participant row for deletion."
+                                                        {% if form.instance.started_at %} disabled {% endif %}
+                                                        @click="
```

Comment:
I’m surprised this isn’t bound on the `change` event, would be worth checking whether this works as expected with keyboard interaction.

Otherwise it would have been nice if there was a way to do this with two-way data binding in Alpine.js, rather than having to write the logic ourselves, but I’m not sure if there would be.

Reply:
Good catch, I changed to `@change`.

Reply:
I also investigated using a built in model method to link checkboxes to the array but couldn't find anything?